components/hidden_data_box/wikis/brawlstars/hidden_data_box_custom.lua

```diff
@@ -0,0 +1,37 @@
+---
+-- @Liquipedia
+-- wiki=brawlstars
+-- page=Module:HiddenDataBox/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Class = require('Module:Class')
+local Variables = require('Module:Variables')
+
+local BasicHiddenDataBox = require('Module:HiddenDataBox')
+local CustomHiddenDataBox = {}
+
+function CustomHiddenDataBox.run(args)
+	BasicHiddenDataBox.addCustomVariables = CustomHiddenDataBox.addCustomVariables
+	return BasicHiddenDataBox.run(args)
+end
+
+function CustomHiddenDataBox.addCustomVariables(args, queryResult)
+	Variables.varDefine('tournament_tier', Variables.varDefault('tournament_liquipediatier'))
+	Variables.varDefine('tournament_tier_type', Variables.varDefault('tournament_liquipediatiertype'))
+	Variables.varDefine('tournament_ticker_name', Variables.varDefault('tournament_tickername'))
+	Variables.varDefine('tournament_parent_name', queryResult.name or '')
+	Variables.varDefine('tournament_riot_premier', Variables.varDefault('tournament_publishertier', ''))
```

Comment:
this var is not required (publisher is supercell) non standard version was probably copied from LOL/Valo and afaik brawlstars never used `|supercellpremier=` in their infoboxes